experiments/mcp/src/pkg/promotion.py

```diff
@@ -270,11 +270,19 @@ def promote_test_fixture(test_fixture_path: str, target_version: str, base_dir:
         source_version = get_version_from_path(abs_test_fixture_path)
         new_test_fixture_path_rel = test_fixture_path.replace(source_version, target_version)
         new_test_fixture_path_abs = to_abs_path(new_test_fixture_path_rel, base_dir)
+        # Move the dir instead of copying to avoid issues with existing directories
+        if os.path.exists(new_test_fixture_path_abs):
+            shutil.rmtree(new_test_fixture_path_abs)
```

Comment:
This will get rid of the "#1" in .goldn.yaml files. It happens when WRITE_GOLDEN_OUTPUT find 2 tests of the same name (One for alpha and one for beta)